src/client/componentsMealsharing/DeleteMeal.js

```diff
@@ -0,0 +1,47 @@
+import "./Meal.css";
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+
+export const DeleteMeal = () => {
+  const [id, setId] = useState();
+  const [success, setSuccess] = useState();
+  
+  const onDeleteMeal =  () => {
+    const inputId = Number(id);
+   fetch(`http://localhost:3000/api/meals/${inputId}`, {
```

Comment:
Could add a check if the id is set before submitting but minor thing :)

Reply:
good suggestion

Reply:
How to check here, Thanks for telling. :)